assets/js/docready.js

```diff
@@ -0,0 +1,77 @@
+(function(funcName, baseObj) {
```

Comment:
@cristian-ungureanu what's the purpose of this file? It seems that is not used anywhere ?

Reply:
It has no purpose,  tried something and forgot it there. Sorry.